components/Slider/src/index.svelte

```diff
@@ -192,9 +193,15 @@
 			style={btnStyle}
 		>
 			{#if $$slots.buttonRender}
-				<slot name="buttonRender" />
+				<KTooltip placement="top" content={String(value)}>
```

Comment:
I think the display of `tooltip` can be changed to an optional prop, which is enabled by default.